flutter/shell/platform/tizen/tizen_renderer.h

```diff
@@ -14,33 +17,21 @@ class TizenRenderer {
   TizenRenderer();
 
   virtual ~TizenRenderer();
+  bool IsValid() { return is_valid_; }
 
+  virtual void ResizeSurface(int32_t width, int32_t height) = 0;
   virtual bool CreateSurface(void* render_target,
                              void* render_target_display,
                              int32_t width,
                              int32_t height) = 0;
```

Comment:
How about moving CreateSurface() and DestroySurface() methods to protected because they are only called by TizenRenderer's children classes?